Tasks/ApiCompat/source/task.json

```diff
@@ -1,8 +1,8 @@
 {
-    "id": "7e15a9bc-91c1-488a-b10e-a8106c6e7523",
-    "name": "binaries-comparer",
-    "friendlyName": "Binaries Comparer",
-    "description": "Binaries Comparer",
+    "id": "fd223c26-a98b-4c0d-976f-99086d7e5ae4",
```

Comment:
Is the new ID needed to avoid clashing with the previous version? Will this new ID be the final one?

Reply:
@JuanAr You're right, those changes shouldn't have been committed. I just actualized the commit without those changes.